lib/VM/JSLib/Math.cpp

```diff
@@ -205,18 +205,26 @@ CallResult<HermesValue> mathPow(void *, Runtime &runtime, NativeArgs args) {
   return HermesValue::encodeUntrustedNumberValue(expOp(x, y));
 }
 
+// hoisting to top-level benchmarks faster than static local
+std::uniform_real_distribution<> _std_real_dist_01(0.0, 1.0);
```

Comment:
In general, we avoid having any global constructors. Keeping this as a static local or a global may also not be thread safe, as multiple runtimes can exist in the same process running on different threads. (`std::uniform_real_distribution<RealType>::operator()` is not const)

We can just keep this as a local to avoid these issues and minimise the scope of this change.

Reply:
Sounds good.